backend/codesirius/problems/views/language.py

```
@@ -0,0 +1,149 @@
+
+    def get(self, request: Request, pk: int) -> CodesiriusAPIResponse:
+        """
+        Get a language by its primary key
+        """
+        logger.info(f"Fetching language with ID: {pk}")
+        try:
+            language = Language.objects.get(pk=pk)
+            logger.info(f"Language with ID: {pk} fetched successfully")
+        except Language.DoesNotExist:
+            logger.warning(f"Language with ID: {pk} not found")
+            raise NotFound(f"Language with ID: {pk} not found")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)